go.sum

```diff
@@ -869,10 +869,16 @@ github.com/vektah/gqlparser v1.1.2 h1:ZsyLGn7/7jDNI+y4SEhI4yAxRChlv15pUHMjijT+e6
 github.com/vektah/gqlparser v1.1.2/go.mod h1:1ycwN7Ij5njmMkPPAOaRFY4rET2Enx7IkVv3vaXspKw=
 github.com/verrazzano/verrazzano-coh-cluster-operator v0.0.8 h1:l1prRSIA+5uVapHXc/j4O1Gf5ZxZPom5Lspg0AF6GJU=
 github.com/verrazzano/verrazzano-coh-cluster-operator v0.0.8/go.mod h1:rBbERv1D+QcHlf7aZVulAOjU3Rv4nHOINZhBmcrt9bg=
-github.com/verrazzano/verrazzano-crd-generator v0.3.31 h1:VuzOfGqm0t4hFTTEhKaRFKXER7XgJGYBAlN7iXVm8xE=
-github.com/verrazzano/verrazzano-crd-generator v0.3.31/go.mod h1:m9coaK+3mJPr+jVWJ18ydpUg/CD659dFxiSshiEudtw=
-github.com/verrazzano/verrazzano-crd-generator v0.3.32 h1:k3ttTXoD5WhnpNiKNu3QxGJb4yfFo23EGWziW5G3V7U=
-github.com/verrazzano/verrazzano-crd-generator v0.3.32/go.mod h1:m9coaK+3mJPr+jVWJ18ydpUg/CD659dFxiSshiEudtw=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33-0.20200827162926-713ae765cd55 h1:ri7h+Fn8TiFipnwaTocdCm91gM3FUtHEqiqnGC+g4Xk=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33-0.20200827162926-713ae765cd55/go.mod h1:m9coaK+3mJPr+jVWJ18ydpUg/CD659dFxiSshiEudtw=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33-0.20200901132156-bc0c2cfddc4c h1:n3eWNFL7B1x6smOl4Q+djw6jznkVMR/0nPMPwfz+Zbc=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33-0.20200901132156-bc0c2cfddc4c/go.mod h1:m9coaK+3mJPr+jVWJ18ydpUg/CD659dFxiSshiEudtw=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33-0.20200901185434-fdba91fff3ba h1:jAUvupn/Q+Hieyij2vAC5LUvazwmfCFJ8vW8QDzM+Eg=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33-0.20200901185434-fdba91fff3ba/go.mod h1:m9coaK+3mJPr+jVWJ18ydpUg/CD659dFxiSshiEudtw=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33-0.20200902212641-e6e120ab187a h1:b2C2dIwYehwvxyKyEcviJ/p/T9B4kFZewoB4iGIuZ0U=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33-0.20200902212641-e6e120ab187a/go.mod h1:m9coaK+3mJPr+jVWJ18ydpUg/CD659dFxiSshiEudtw=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33 h1:RIcInQTJqKzREph8MwWccxTXyzGnPa1lCskVQzy8kmk=
+github.com/verrazzano/verrazzano-crd-generator v0.3.33/go.mod h1:m9coaK+3mJPr+jVWJ18ydpUg/CD659dFxiSshiEudtw=
 github.com/verrazzano/verrazzano-helidon-app-operator v0.0.7 h1:MkkMH/WNPHEDYElmEE6wBZKaNGZIhXdi2FWKpsgEnl8=
```

Comment:
odd why so many entries got added to go.sum for v0.3.33.  May want to try deleting go.sum and rebuild it with `make go-mod`

Reply:
I generated a new go.sum